chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/activityMiddleware.tsx

```diff
@@ -46,11 +46,9 @@ const handleSystemMessage = (next: any, args: any[], card: any, systemMessageSty
         return () => false;
     }
 
-    // eslint-disable-next-line react/display-name, @typescript-eslint/no-explicit-any
+    // eslint-disable-next-line react/display-name
     return () => (
-        <div key={card.activity.id} style={systemMessageStyles} aria-hidden="true">
-            {card.activity.text}
-        </div>
+        <div key={card.activity.id} style={systemMessageStyles} aria-hidden="true" dangerouslySetInnerHTML={{ __html: card.activity.text }}/>
```

Comment:
Can it raise any Security Risk ?
For e.g.,
dangerouslySetInnerHTML={{ __html: content }}
content = '<img src=x onerror="alert(\'XSS Attack\')">';

Reply:
Why is this a security risk?

Reply:
![image](https://github.com/microsoft/omnichannel-chat-widget/assets/14852927/dc29bb92-2987-4be5-969d-04c7abc44c6f)

Reply:
I can repro the attack. But since the message can only be added by admin, is there need to check for script tags?

Reply:
Couple of things 

1. Does Admin allow to add the malicious content? If yes, we should check with them if this should be a bug at their end or what is their expectation if they are allowing this.
2. From LCW side, I still think, we should take care of XSS attacks in all such type of fixes.

Let's check Naresh's point of view on this

Reply:
Actually nvm. Used the existing `escapeHtml` function
![image](https://github.com/microsoft/omnichannel-chat-widget/assets/14852927/4aa53121-4704-4d08-9cc4-a8be48a92ae1)
![image](https://github.com/microsoft/omnichannel-chat-widget/assets/14852927/f02ba7ba-4eb8-4239-89af-257f2462c09f)